models/user.js

```diff
@@ -30,7 +30,8 @@ const UserSchema = mongoose.Schema({
     type: Boolean,
     default: false
   },
-  edit_token: String
+  edit_token: String,
+  followed_cubes: [Number]
```

Comment:
maybe call this `followed_cube_ids` to be more descriptive of what the data actually is